lib/theme/index.tsx

```diff
@@ -190,20 +204,27 @@ theme = createTheme({
           fontFamily: primaryFont,
           borderRadius: '1rem',
           padding: '0.125rem 0.5rem',
+          backgroundColor: gray100,
+          color: gray700
         },
         label: {
           fontSize: '0.75rem',
           lineHeight: '150%',
           fontWeight: 500,
           padding: 0,
           fontFamily: primaryFont,
+          paddingLeft: '0.25rem'
```

Comment:
@Aiga115 instead of padding left on the label you should instead use a gap on parent, because if we dont use an icon here in the chip, there is still going to be space in the left